### PR TITLE
[PFD] Add altitude C chord warning sounds

### PR DESCRIPTION
**Pull request by ilyeshammadi**
## Summary of Changes

This PR is a succession of this PR #1889 and adds the C chords sound when deviating from the selected altitude.
**What has been done**

- [x] Trigger C chords sound when in Yellow/Amber scenario.
- [ ] Stop the triggered sound by clicking on MASTER CAUT or MASTER WARN.

## Additional context

Discord username (if different from GitHub): Union Araken#4083

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml [hunk 1]
```diff
@@ -882,6 +882,21 @@
             </Requires>
         </Sound>
 
+        <!-- Altitude deviation warning =====================================================================-->
+                <Sound WwiseData="true" WwiseEvent="cchordloop" NodeName="WIPER_BASE_L" ViewPoint="Inside" LocalVar="A32NX_ALT_DEVIATION" Continuous="true">
```
Comment (ilyeshammadi) [inline A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml:886]: @hotshotp to confirm the XML sound structure.

**Comment by Kimbyeoungjang** [comment]
oh you working this. i'll close my draft PR #2066. thank you for your working :)

**Comment by ilyeshammadi** [comment]
> oh you working this. i'll close my draft PR #2066. thank you for your working :)

Yes, was waiting for progress on your PR, but then decided to continue on my own. Thanks again for your work 😄

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml (hunk 1 above)
Comment (hotshotp) [reply A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml:886]: Node name should be PEDALS_LEFT other wise good

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml [hunk 2]
```diff
@@ -882,6 +882,21 @@
             </Requires>
         </Sound>
 
+        <!-- Altitude deviation warning =====================================================================-->
```
Comment (Benjozork) [inline A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml:885]: Indentation

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/AltimeterIndicator.js [hunk 3]
```diff
@@ -461,15 +461,22 @@ class Jet_PFD_AltimeterIndicator extends HTMLElement {
 
     _pulseYellow() {
         this.cursorSVGShape.classList.add("pulse-yellow");
+        // Trigger the warning sound only if both auto pilots are disabled
```
Comment (Benjozork) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/AltimeterIndicator.js:464]: As discussed on Discord we should find a way to move this out of the PFD

**Comment by Kimbyeoungjang** [comment]
maybe you have some problem with cancel warning( use warning or caution button, stop the sound)
because i also have that problem. my one of pr is A/THR disconnect warning. that also stop the light when press warning or caution but current status is not.

maybe we can work this with same code(about warning and caution system)

**Comment by hotshotp** [comment]
how is this coming along?

**Pull request closed without merging**